### ISSUE TITLE
review tutorial and demo provided by ai-on-openshift

### ISSUE THREAD

**Issue by goern** [operate-first/genAI-for-document-generation#1]
As a team member, 
I want to get an overview of the solution and the demo and the demo flow, 
so that I can explain it to others, and so that I have a clear view of what this is all about.

- [ ] https://issues.redhat.com/browse/APPENG-2038
- [ ] https://issues.redhat.com/browse/APPENG-2064
- [ ] https://issues.redhat.com/browse/APPENG-2080

**Comment by swati-kale** [comment]
> @swati-kale the upstream work is at https://github.com/rh-aiservices-bu/llm-on-openshift/tree/main/examples/ui

Yes , we are working on respective changes using upstream baseline
